core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -1413,9 +1405,10 @@ func TestLogPoller_DBErrorHandling(t *testing.T) {
 	time.Sleep(100 * time.Millisecond)
 	require.NoError(t, lp.Start(ctx))
 	require.Eventually(t, func() bool {
-		return observedLogs.Len() >= 4
+		return observedLogs.Len() >= 2
```

Comment:
Which change affected this ?

Reply:
I think not using the pg SQL logging changed this, and different PR reduced this number as well.